resources/authgear/templates/en/web/authflowv2/change_password_success.html

```diff
@@ -0,0 +1,18 @@
+{{ template "authflowv2/__page_frame.html" . }}
+{{ define "page-content" }}
+<div class="flex-1-0-auto screen-icon-layout tablet:flex-none" >
+  <i class="material-icons screen-icon">check_circle</i>
+  <header class="flex flex-col gap-4">
+    <h1 class="screen-title mt-2">
+      {{ template "v2-change-password-success-page-title" }}
+    </h1>
+    <p class="screen-description">{{ template "v2-change-password-success-page-description" }}</p>
+  </header>
+  <div></div>
+    <footer>
+      <a class="primary-btn w-full" href="{{ call $.MakeURL "/settings" "x_step" "" "x_can_back_to_login" "" }}" data-turbo="false">
+        {{ template "v2-button.continue" }}
+      </a>
```

Comment:
I'm not sure should we hardcode `/settings` here. Referencing the interaction webapp it seems we didn't hardcode the redirect uri here.

Reply:
Should go to finish route but forgot updating it.

How about hardcoding finish route / send post request to navigate to finish route?

Reply:
It might not be finish route if the flow is not finished, so we cannot hardcode it here. We need to send a post request to the server, and let the server to decide where to go, which is what `AdvanceWithoutInput` should do.